media-gfx/zbar/zbar-0.23.ebuild

```diff
@@ -0,0 +1,203 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit autotools flag-o-matic java-pkg-opt-2 multilib-minimal python-single-r1 virtualx
+
+DESCRIPTION="Library and tools for reading barcodes from images or video"
+HOMEPAGE="https://github.com/mchehab/zbar"
+SRC_URI="https://linuxtv.org/downloads/${PN}/${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+
+IUSE="dbus graphicsmagick gtk gtk2 imagemagick introspection java jpeg png python qt5 static-libs test +threads v4l X xv"
+REQUIRED_USE="
+	introspection? (
+		|| ( gtk gtk2 )
+	)
+	python? ( ${PYTHON_REQUIRED_USE} )
+	qt5? ( X )
+	test? (
+		|| ( graphicsmagick imagemagick )
+		${PYTHON_REQUIRED_USE}
+		X png
+	)"
+
+RESTRICT="!test? ( test )"
+
+COMMON_DEPEND="
+	dbus? ( sys-apps/dbus[${MULTILIB_USEDEP}] )
+	gtk? (
+		dev-libs/glib:2[${MULTILIB_USEDEP}]
+		x11-libs/gtk+:3[${MULTILIB_USEDEP}]
+	)
+	gtk2? (
+		dev-libs/glib:2[${MULTILIB_USEDEP}]
+		x11-libs/gtk+:2[${MULTILIB_USEDEP}]
+	)
+	graphicsmagick? ( media-gfx/graphicsmagick:= )
+	imagemagick? ( media-gfx/imagemagick:= )
```

Comment:
what happens with USE="graphicsmagick imagemagick"?

Reply:
it use ImageMagick by default

Reply:
configure results:
```bash
please verify that the detected configuration matches your expectations:
------------------------------------------------------------------------
X                      --with-x=yes
pthreads               --enable-pthread=yes
doc                    --enable-doc=yes
v4l                    --enable-video=yes
jpeg                   --with-jpeg=yes
Python                 --with-python=no    python
GTK                    --with-gtk=gtk3          Gtk3.24.11
GObject introspection  --with-gir=yes
Qt                     --with-qt=yes            Qt5.13.2
Java                   --with-java=yes
Dbus                   --with-dbus=yes
ImageMagick            --with-imagemagick=yes
Enabled codes:         ean databar code128 code93 code39 codabar i25 qrcode sqcode
Disabled codes:        pdf417
JAVA_HOME              /opt/icedtea-bin-3.14.0
```

Reply:
>Python                 --with-python=xno    python3.6

hm... one momend

Reply:
fixed: Python                 --with-python=auto    python3.6

Reply:
> it use ImageMagick by default

okay. we are solving this ffmpeg/libav style, please have a look at `inkscape-1.0_beta2.ebuild` for how to do that.